packages/lib/src/components/internal/Await/Await.tsx

```diff
@@ -106,15 +106,7 @@ function Await(props: AwaitComponentProps) {
     };
 
     useEffect(() => {
-        const { shouldRedirectOnMobile, url } = props;
```

Comment:
[shouldRedirectOnMobile](https://github.com/Adyen/adyen-web/blob/4d55ec1349a83fb6694238c2eebb2a16f47a7add/packages/lib/src/components/internal/Await/Await.tsx#L223)  still defined as defaultProps in the `Await.defaultProps = {...` . It should be removed